x-pack/plugins/security_solution/server/usage/detections.mocks.ts

```diff
@@ -0,0 +1,162 @@
+    groups: ['auditbeat', 'process', 'siem'],
+    processed_record_count: 0,
+    memory_status: 'ok',
+    jobState: 'closed',
+    hasDatafeed: true,
+    datafeedId: 'datafeed-linux_anomalous_network_port_activity_ecs',
+    datafeedIndices: ['auditbeat-*'],
+    datafeedState: 'stopped',
+    isSingleMetricViewerJob: true,
+  },
+  {
+    id: 'other_job',
+    description: 'a job that is custom',
+    groups: ['auditbeat', 'process'],
+    processed_record_count: 0,
+    memory_status: 'ok',
+    jobState: 'closed',
+    hasDatafeed: true,
+    datafeedId: 'datafeed-other',
+    datafeedIndices: ['auditbeat-*'],
+    datafeedState: 'stopped',
+    isSingleMetricViewerJob: true,
+  },
+  {
+    id: 'another_job',
+    description: 'another job that is custom',
+    groups: ['auditbeat', 'process'],
+    processed_record_count: 0,
+    memory_status: 'ok',
+    jobState: 'opened',
+    hasDatafeed: true,
+    datafeedId: 'datafeed-another',
+    datafeedIndices: ['auditbeat-*'],
+    datafeedState: 'started',
+    isSingleMetricViewerJob: true,
+  },
+];
+
+export const getMockListModulesResponse = () => [
+  {
+    id: 'siem_auditbeat',
+    title: 'SIEM Auditbeat',
+    description:
+      'Detect suspicious network activity and unusual processes in Auditbeat data (beta).',
+    type: 'Auditbeat data',
+    logoFile: 'logo.json',
+    defaultIndexPattern: 'auditbeat-*',
+    query: {
+      bool: {
+        filter: [
+          {
+            term: {
+              'agent.type': 'auditbeat',
+            },
+          },
+        ],
+      },
+    },
+    jobs: [
+      {
+        id: 'linux_anomalous_network_activity_ecs',
+        config: {
+          job_type: 'anomaly_detector',
+          description:
+            'SIEM Auditbeat: Looks for unusual processes using the network which could indicate command-and-control, lateral movement, persistence, or data exfiltration activity (beta)',
+          groups: ['siem', 'auditbeat', 'process'],
+          analysis_config: {
+            bucket_span: '15m',
+            detectors: [
+              {
+                detector_description: 'rare by "process.name"',
+                function: 'rare',
+                by_field_name: 'process.name',
+              },
+            ],
+            influencers: ['host.name', 'process.name', 'user.name', 'destination.ip'],
+          },
+          allow_lazy_open: true,
+          analysis_limits: {
+            model_memory_limit: '64mb',
+          },
+          data_description: {
+            time_field: '@timestamp',
+          },
+        },
+      },
+      {
+        id: 'linux_anomalous_network_port_activity_ecs',
+        config: {
+          job_type: 'anomaly_detector',
+          description:
+            'SIEM Auditbeat: Looks for unusual destination port activity that could indicate command-and-control, persistence mechanism, or data exfiltration activity (beta)',
```

Comment:
Just checking we haven't migrated these detections to the `SecuritySolution` naming pattern?

Reply:
I took these from live data and the rename hasn't happened yet. I'll be sure to update these appropriately once that happens 👍